style.css

```diff
@@ -652,3 +661,820 @@ form ul {
   right: 0.5%;
   margin-top: 35px;
 }
+
+.portfolio {
+  display: none;
+}
+
+.contact3 {
+  display: none;
+}
+
+.about2 {
+  display: none;
+}
+
+.menu-media {
+  display: none;
+}
+
+.imag-tonic1 {
+  display: none;
+}
+
+.text-mypost-hid {
+  display: none;
+}
+
+.text-mypost-orig {
+  display: flex;
+}
+
+#full-stack {
+  display: none;
+}
+
+#full-stack1 {
+  display: none;
+}
+
+#ruby {
+  display: none;
+}
+
+#ruby1 {
+  display: none;
+}
+
+.imag-mypost2 {
+  display: none;
+}
+
+#facebook {
+  display: none;
+  color: #344563;
+  font-weight: 600;
+}
+
+#facebook1 {
+  display: none;
+  color: #344563;
+  font-weight: 600;
+}
+
+.imag-tonic-4 {
+  display: none;
+}
+
+.title-tonic2 {
+  display: none;
+}
+
+.imag-tonic-6 {
+  display: none;
+}
+
+.Title-avility2 {
+  display: none;
+}
+
+#uber {
+  display: none;
+}
+
+#year2 {
+  display: none;
+}
+
+#lead {
+  display: none;
+}
+
+.text-mypost-last1 {
+  display: none;
+}
+
+#java1 {
+  display: none;
+}
+
+#html1 {
+  display: none;
+}
+
+#css1 {
+  display: none;
+}
+
+#frame1 {
+  display: none;
+}
+
+#ski1 {
+  display: none;
+}
+
+.list-complete li {
+  display: inline;
+}
+
+.Q {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  color: #253858;
+  margin-left: -50px;
+  margin-bottom: 30px;
+  background: #f7f7f9;
+  margin-top: 20px;
+}
+
+.Q p {
+  margin-left: 20px;
+}
+
+.languages {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  margin-left: -20px;
+}
+
+.arrow {
+  position: absolute;
+  right: 13%;
+  margin-top: -50px;
+}
+
+.arrow2 {
+  height: 4%;
+  position: absolute;
+  right: 13%;
+  margin-top: -58px;
+}
+
+.arrow3 {
+  height: 4%;
+  position: absolute;
+  right: 13%;
+  margin-top: -58px;
+}
+
+.languajes {
+  position: relative;
+}
+
+.languages2 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  margin-left: -20px;
+  margin-top: 50px;
+}
+
+.separator1 {
+  width: 95%;
+  height: 1px;
+  background: #dfe1e6;
+  margin-left: -2%;
+  margin-top: 30px;
+}
+
+.separator2 {
+  width: 95%;
+  height: 1px;
+  background: #dfe1e6;
+  margin-left: -2%;
+  margin-top: 30px;
+}
+
+.list-complete {
+  display: none;
+}
+
+@media (min-width: 728px) {
+  body {
+    overflow-x: hidden;
+  }
+
+  .fondoo {
+    display: none;
+  }
+
+  .fondoo1 {
+    display: flex;
+  }
+
+  .logo {
+    padding-left: 150px;
+  }
+
+  .contenedor {
```

Comment:
[OPTIONAL] You are doing great, just a friendly piece of advice, try to select one language for your project to avoid mixing classes in Spanish with classes in English, this way your code will be easier to maintain by other developers.